encodings/fastlanes/src/bitpacking/compress.rs

```diff
@@ -20,6 +20,13 @@ pub fn bitpack_encode(array: PrimitiveArray, bit_width: u8) -> VortexResult<BitP
         .statistics()
         .compute_bit_width_freq()
         .ok_or_else(|| vortex_err!(ComputeError: "missing bit width frequency"))?;
+    let has_negative_values = match_each_integer_ptype!(array.ptype(), |$P| {
+        array.statistics().compute_min::<$P>().unwrap_or_default() < 0
```

Comment:
only need to do this if it's a signed int, otherwise we don't need to compute min at all